package.json

```diff
@@ -88,6 +88,7 @@
     "angular-translate": "2.13.1",
     "ansicolors": "0.3.2",
     "autoprefixer": "6.5.4",
+    "axios": "0.17.0",
```

Comment:
I posted a question in the kibana slack channel and got the thumbs up for axios

Reply:
That's good to know, but I still think a Github issue would be worthwhile. Slack isn't great for making decisions with far reaching consequences since everyone who might have an opinion might not be watching Slack at that exact moment. I'm particularly paranoid in this instance since we were just talking in Berlin about how we need to be more careful about brining in new dependencies. If you'd prefer, I can do the legwork of filing an issue and making everyone aware of it.